airflow/providers/vertica/hooks/vertica.py

```diff
@@ -45,6 +45,51 @@ def get_conn(self) -> connect:
             conn_config["port"] = 5433
         else:
             conn_config["port"] = int(conn.port)
+            
+        if conn.extra_dejson.get("connection_load_balance", False):
+            conn_config["connection_load_balance"] = bool(conn.extra_dejson["connection_load_balance"])
+            
+        if conn.extra_dejson.get("session_label", False):
+            conn_config["session_label"] = conn.extra_dejson["session_label"]
 
+        if conn.extra_dejson.get("backup_server_node", False):
+            conn_config["backup_server_node"] = conn.extra_dejson["backup_server_node"]
+
+        if conn.extra_dejson.get("binary_transfer", False):
+            conn_config["binary_transfer"] = bool(conn.extra_dejson["binary_transfer"])
+
+        if conn.extra_dejson.get("connection_timeout", False):
+            conn_config["connection_timeout"] = int(conn.extra_dejson["connection_timeout"])
+
+        if conn.extra_dejson.get("disable_copy_local", False):
+            conn_config["disable_copy_local"] = bool(conn.extra_dejson["disable_copy_local"])
+
+        if conn.extra_dejson.get("kerberos_host_name", False):
+            conn_config["kerberos_host_name"] = conn.extra_dejson["kerberos_host_name"]
+
+        if conn.extra_dejson.get("kerberos_service_name", False):
+            conn_config["kerberos_service_name"] = conn.extra_dejson["kerberos_service_name"]
+
+        if conn.extra_dejson.get("log_level", False):
```

Comment:
is `False` the right default value for `log_level`. From what I can see from  [here](https://github.com/vertica/vertica-python#logging), it seems to be _logging.WARNING_.

The same question for `log_path`

Reply:
the .get(<option_name>, False) is not used for declare a default value but for test the presence of the option key in extra dictionnary.

Reply:
mb seems like i presumed something i shouldn't. Looks good here!